### ISSUE TITLE
poetry install fails with [ValueError] No file/folder found for package

### ISSUE THREAD

**Issue by lmk-cc** [sdispater/poetry#512]
- [x] I am on the [latest](https://github.com/sdispater/poetry/releases/latest) Poetry version.
- [x] I have searched the [issues](https://github.com/sdispater/poetry/issues) of this repo and believe that this is not a duplicate.
- [x] If an exception occurs when executing a command, I executed it again in debug mode (`-vvv` option).

- **OS version and name**: Ubuntu 14.04
- **Poetry version**: 0.12.2
- **Link of a [Gist](https://gist.github.com/) with the contents of your pyproject.toml file**: https://gist.github.com/lmk-cc/a3f9268904ab45049360f2ccf378fee4

## Issue

Hi there, we are having issues with the latest version of poetry on CircleCI.

Here is the relevant section from the circle config:

```
      - run:
          name: install dependencies
          command: |
            curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
            source $HOME/.poetry/env
            python -m venv .venv
            source .venv/bin/activate
            poetry config settings.virtualenvs.in-project true
            make install
```

The `make install` target runs `poetry install -vvv`.

It doesn't complete, and I get the following output:

```
#!/bin/bash -eo pipefail
curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
source $HOME/.poetry/env
python -m venv .venv
source .venv/bin/activate
poetry config settings.virtualenvs.in-project true
make install

Retrieving Poetry metadata

# Welcome to Poetry!

This will download and install the latest version of Poetry,
a dependency and package manager for Python.

It will add the `poetry` command to Poetry's bin directory, located at:

$HOME/.poetry/bin

This path will then be added to your `PATH` environment variable by
modifying the profile file located at:

$HOME/.profile

You can uninstall at any time with `poetry self:uninstall`,
or by executing this script with the --uninstall option,
and these changes will be reverted.

Installing version: 0.12.2
  - Downloading poetry-0.12.2-linux.tar.gz (8.53MB)

Poetry (0.12.2) is installed now. Great!

To get started you need Poetry's bin directory ($HOME/.poetry/bin) in your `PATH`
environment variable. Next time you log in this will be done
automatically.

To configure your current shell run `source $HOME/.poetry/env`

Using virtualenv: /home/circleci/repo/.venv
Updating dependencies
Resolving dependencies...
   1: fact: mos-locate-service is 0.0.1
   1: derived: mos-locate-service
   1: fact: mos-locate-service depends on django (^2.0)
   1: fact: mos-locate-service depends on dj_database_url (^0.5.0)
   1: fact: mos-locate-service depends on envparse (^0.2.0)
   1: fact: mos-locate-service depends on psycopg2-binary (^2.7)
   1: fact: mos-locate-service depends on pyjwt (^1.6)
   1: fact: mos-locate-service depends on django_grpc (*)
   1: fact: mos-locate-service depends on pytz (^2018.5)
   1: fact: mos-locate-service depends on requests (^2.19)
   1: fact: mos-locate-service depends on pillow (^5.3)
   1: fact: mos-locate-service depends on pytest (^3.0)
   1: fact: mos-locate-service depends on pytest-django (^3.3)
   1: fact: mos-locate-service depends on doc8 (^0.8.0)
   1: fact: mos-locate-service depends on Pygments (^2.2)
   1: fact: mos-locate-service depends on black (^18.3-alpha.0)
   1: fact: mos-locate-service depends on isort (^4.3)
   1: fact: mos-locate-service depends on mypy (^0.620.0)
   1: fact: mos-locate-service depends on pytest-cov (^2.5)
   1: fact: mos-locate-service depends on pytest-spec (^1.1)
   1: fact: mos-locate-service depends on ipython (^6.5)
   1: fact: mos-locate-service depends on factory_boy (^2.11)
   1: fact: mos-locate-service depends on pudb (^2018.1)
   1: fact: mos-locate-service depends on mypy-protobuf (^1.6)
   1: fact: mos-locate-service depends on pyflakes (^2.0)
   1: selecting mos-locate-service (0.0.1)
   1: derived: pyflakes (^2.0)
   1: derived: mypy-protobuf (^1.6)
   1: derived: pudb (^2018.1)
   1: derived: factory_boy (^2.11)
   1: derived: ipython (^6.5)
   1: derived: pytest-spec (^1.1)
   1: derived: pytest-cov (^2.5)
   1: derived: mypy (^0.620.0)
   1: derived: isort (^4.3)
   1: derived: black (^18.3-alpha.0)
   1: derived: Pygments (^2.2)
   1: derived: doc8 (^0.8.0)
   1: derived: pytest-django (^3.3)
   1: derived: pytest (^3.0)
   1: derived: pillow (^5.3)
   1: derived: requests (^2.19)
   1: derived: pytz (^2018.5)
   1: derived: django_grpc (*)
   1: derived: pyjwt (^1.6)
   1: derived: psycopg2-binary (^2.7)
   1: derived: envparse (^0.2.0)
   1: derived: dj_database_url (^0.5.0)
   1: derived: django (^2.0)
PyPI: No release information found for pyflakes-0.2.0, skipping
PyPI: No release information found for pyflakes-0.2.1, skipping
PyPI: 1 packages found for pyflakes >=2.0,<3.0
PyPI: 2 packages found for mypy-protobuf >=1.6,<2.0
PyPI: 1 packages found for pudb >=2018.1,<2019.0
PyPI: 2 packages found for factory-boy >=2.11,<3.0
PyPI: No release information found for ipython-0.6.10, skipping
PyPI: No release information found for ipython-0.6.11, skipping
PyPI: No release information found for ipython-0.6.12, skipping
PyPI: No release information found for ipython-0.6.13, skipping
PyPI: No release information found for ipython-0.6.14, skipping
PyPI: No release information found for ipython-0.6.15, skipping
PyPI: No release information found for ipython-0.6.4, skipping
PyPI: No release information found for ipython-0.6.5, skipping
PyPI: No release information found for ipython-0.6.6, skipping
PyPI: No release information found for ipython-0.6.7, skipping
PyPI: No release information found for ipython-0.6.8, skipping
PyPI: No release information found for ipython-0.6.9, skipping
PyPI: No release information found for ipython-0.7.0, skipping
PyPI: No release information found for ipython-0.7.1, skipping
PyPI: No release information found for ipython-0.7.1.fix1, skipping
PyPI: No release information found for ipython-0.7.2, skipping
PyPI: No release information found for ipython-0.7.4.svn.r2010, skipping
PyPI: No release information found for ipython-0.8.0, skipping
PyPI: No release information found for ipython-0.8.1, skipping
PyPI: No release information found for ipython-0.8.2, skipping
PyPI: No release information found for ipython-0.8.3, skipping
PyPI: No release information found for ipython-0.8.4, skipping
PyPI: No release information found for ipython-0.9, skipping
PyPI: No release information found for ipython-0.9.1, skipping
PyPI: No release information found for ipython-4.0.0-b1, skipping
PyPI: 3 packages found for ipython >=6.5,<7.0
PyPI: 1 packages found for pytest-spec >=1.1,<2.0
PyPI: 3 packages found for pytest-cov >=2.5,<3.0
PyPI: No release information found for mypy-0.500, skipping
PyPI: 1 packages found for mypy >=0.620.0,<0.621.0
PyPI: 5 packages found for isort >=4.3,<5.0
PyPI: 18 packages found for black >=18.3-alpha.0,<19.0
PyPI: 1 packages found for pygments >=2.2,<3.0
PyPI: 1 packages found for doc8 >=0.8.0,<0.9.0
PyPI: 6 packages found for pytest-django >=3.3,<4.0
PyPI: 40 packages found for pytest >=3.0,<4.0
PyPI: 1 packages found for pillow >=5.3,<6.0
PyPI: No release information found for requests-0.0.1, skipping
PyPI: No release information found for requests-0.12.01, skipping
PyPI: No release information found for requests-2.15.0, skipping
PyPI: 2 packages found for requests >=2.19,<3.0
PyPI: 1 packages found for pytz >=2018.5,<2019.0
PyPI: 4 packages found for pyjwt >=1.6,<2.0
PyPI: 3 packages found for psycopg2-binary >=2.7,<3.0
PyPI: 1 packages found for envparse >=0.2.0,<0.3.0
PyPI: No release information found for dj-database-url-0, skipping
PyPI: 1 packages found for dj-database-url >=0.5.0,<0.6.0
PyPI: No release information found for django-1.0.1, skipping
PyPI: No release information found for django-1.0.2, skipping
PyPI: No release information found for django-1.0.3, skipping
PyPI: No release information found for django-1.0.4, skipping
PyPI: No release information found for django-1.1, skipping
PyPI: No release information found for django-1.1.1, skipping
PyPI: No release information found for django-1.1.2, skipping
PyPI: 16 packages found for django >=2.0,<3.0
PyPI: Getting info for pyflakes (2.0.0) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading wheel: pyflakes-2.0.0-py2.py3-none-any.whl
   1: selecting pyflakes (2.0.0)
PyPI: Getting info for pudb (2018.1) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading sdist: pudb-2018.1.tar.gz
   1: fact: pudb (2018.1) depends on urwid (>=1.1.1)
   1: fact: pudb (2018.1) depends on pygments (>=1.0)
   1: selecting pudb (2018.1)
   1: derived: urwid (>=1.1.1)
PyPI: No release information found for urwid-0.8.0, skipping
PyPI: No release information found for urwid-0.8.1, skipping
PyPI: No release information found for urwid-0.8.10, skipping
PyPI: No release information found for urwid-0.8.2, skipping
PyPI: No release information found for urwid-0.8.3, skipping
PyPI: No release information found for urwid-0.8.4, skipping
PyPI: No release information found for urwid-0.8.5, skipping
PyPI: No release information found for urwid-0.8.6, skipping
PyPI: No release information found for urwid-0.8.7, skipping
PyPI: No release information found for urwid-0.8.8, skipping
PyPI: No release information found for urwid-0.8.9, skipping
PyPI: No release information found for urwid-0.9.0, skipping
PyPI: No release information found for urwid-0.9.0-pre2, skipping
PyPI: No release information found for urwid-0.9.0-pre3, skipping
PyPI: No release information found for urwid-0.9.1, skipping
PyPI: No release information found for urwid-0.9.2, skipping
PyPI: No release information found for urwid-0.9.3, skipping
PyPI: No release information found for urwid-0.9.4, skipping
PyPI: No release information found for urwid-0.9.5, skipping
PyPI: No release information found for urwid-0.9.6, skipping
PyPI: No release information found for urwid-0.9.7, skipping
PyPI: No release information found for urwid-0.9.7.2, skipping
PyPI: No release information found for urwid-0.9.8, skipping
PyPI: No release information found for urwid-0.9.8.2, skipping
PyPI: No release information found for urwid-0.9.8.3, skipping
PyPI: No release information found for urwid-0.9.8.4, skipping
PyPI: No release information found for urwid-0.9.9, skipping
PyPI: No release information found for urwid-0.9.9.1, skipping
PyPI: No release information found for urwid-0.9.9.2, skipping
PyPI: No release information found for urwid-1.0.0, skipping
PyPI: No release information found for urwid-1.0.1, skipping
PyPI: 9 packages found for urwid >=1.1.1
PyPI: Getting info for pytest-spec (1.1.0) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading sdist: pytest-spec-1.1.0.tar.gz
   1: fact: pytest-spec (1.1.0) depends on mock (>=1.0.1)
   1: selecting pytest-spec (1.1.0)
   1: derived: mock (>=1.0.1)
PyPI: No release information found for mock-1.0.2.dev57, skipping
PyPI: 9 packages found for mock >=1.0.1
PyPI: Getting info for mypy (0.620) from PyPI
   1: fact: mypy (0.620) depends on typed-ast (>=1.1.0,<1.2.0)
   1: selecting mypy (0.620)
   1: derived: typed-ast (>=1.1.0,<1.2.0)
PyPI: 1 packages found for typed-ast >=1.1.0,<1.2.0
PyPI: Getting info for pygments (2.2.0) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading wheel: Pygments-2.2.0-py2.py3-none-any.whl
   1: selecting pygments (2.2.0)
PyPI: Getting info for doc8 (0.8.0) from PyPI
   1: fact: doc8 (0.8.0) depends on chardet (*)
   1: fact: doc8 (0.8.0) depends on docutils (*)
   1: fact: doc8 (0.8.0) depends on restructuredtext-lint (>=0.7)
   1: fact: doc8 (0.8.0) depends on six (*)
   1: fact: doc8 (0.8.0) depends on stevedore (*)
   1: selecting doc8 (0.8.0)
   1: derived: stevedore (*)
   1: derived: six (*)
   1: derived: restructuredtext-lint (>=0.7)
   1: derived: docutils (*)
   1: derived: chardet (*)
PyPI: 56 packages found for stevedore *
PyPI: 23 packages found for six *
PyPI: 30 packages found for restructuredtext-lint >=0.7
PyPI: 19 packages found for docutils *
PyPI: 11 packages found for chardet *
PyPI: Getting info for pillow (5.3.0) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading sdist: Pillow-5.3.0.tar.gz
PyPI: Downloading wheel: Pillow-5.3.0-cp27-cp27m-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
   1: selecting pillow (5.3.0)
PyPI: Getting info for pytz (2018.5) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading wheel: pytz-2018.5-py2.py3-none-any.whl
   1: selecting pytz (2018.5)
   1: fact: django-grpc (0.1.1) depends on grpcio (*)
   1: fact: django-grpc (0.1.1) depends on grpcio-reflection (*)
   1: fact: django-grpc (0.1.1) depends on pyjwt (*)
   1: fact: django-grpc (0.1.1) depends on django (*)
   1: fact: django-grpc (0.1.1) depends on cryptography (*)
   1: fact: django-grpc (0.1.1) depends on protobuf (*)
   1: selecting django-grpc (0.1.1 7a5009d)
   1: derived: protobuf (*)
   1: derived: cryptography (*)
   1: derived: grpcio-reflection (*)
   1: derived: grpcio (*)
PyPI: 32 packages found for protobuf *
PyPI: 65 packages found for cryptography *
PyPI: 55 packages found for grpcio-reflection *
PyPI: No release information found for grpcio-0.12.0b8, skipping
PyPI: No release information found for grpcio-0.3.0, skipping
PyPI: No release information found for grpcio-0.4.0, skipping
PyPI: 93 packages found for grpcio *
PyPI: Getting info for envparse (0.2.0) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading sdist: envparse-0.2.0.tar.gz
   1: selecting envparse (0.2.0)
PyPI: Getting info for dj-database-url (0.5.0) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading wheel: dj_database_url-0.5.0-py2.py3-none-any.whl
   1: selecting dj-database-url (0.5.0)
PyPI: Getting info for typed-ast (1.1.0) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading sdist: typed-ast-1.1.0.tar.gz
PyPI: Downloading wheel: typed_ast-1.1.0-cp33-cp33m-manylinux1_i686.whl
   1: selecting typed-ast (1.1.0)
PyPI: Getting info for mypy-protobuf (1.7) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading sdist: mypy-protobuf-1.7.tar.gz
   1: selecting mypy-protobuf (1.7)
PyPI: Getting info for factory-boy (2.11.1) from PyPI
   1: fact: factory-boy (2.11.1) depends on Faker (>=0.7.0)
   1: selecting factory-boy (2.11.1)
   1: derived: Faker (>=0.7.0)
PyPI: 40 packages found for faker >=0.7.0
PyPI: Getting info for requests (2.19.1) from PyPI
   1: fact: requests (2.19.1) depends on certifi (>=2017.4.17)
   1: fact: requests (2.19.1) depends on urllib3 (>=1.21.1,<1.24)
   1: fact: requests (2.19.1) depends on idna (>=2.5,<2.8)
   1: fact: requests (2.19.1) depends on chardet (>=3.0.2,<3.1.0)
   1: selecting requests (2.19.1)
   1: derived: chardet (>=3.0.2,<3.1.0)
   1: derived: idna (>=2.5,<2.8)
   1: derived: urllib3 (>=1.21.1,<1.24)
   1: derived: certifi (>=2017.4.17)
PyPI: No release information found for idna-0.1, skipping
PyPI: 3 packages found for idna >=2.5,<2.8
PyPI: No release information found for urllib3-0.3, skipping
PyPI: No release information found for urllib3-0.3.1, skipping
PyPI: No release information found for urllib3-0.4.0, skipping
PyPI: No release information found for urllib3-0.4.1, skipping
PyPI: 3 packages found for urllib3 >=1.21.1,<1.24
PyPI: No release information found for certifi-0, skipping
PyPI: 9 packages found for certifi >=2017.4.17
PyPI: Getting info for ipython (6.5.0) from PyPI
   1: fact: ipython (6.5.0) depends on colorama (*)
   1: fact: ipython (6.5.0) depends on appnope (*)
   1: fact: ipython (6.5.0) depends on pexpect (*)
   1: fact: ipython (6.5.0) depends on backcall (*)
   1: fact: ipython (6.5.0) depends on pygments (*)
   1: fact: ipython (6.5.0) depends on prompt-toolkit (>=1.0.15,<2.0.0)
   1: fact: ipython (6.5.0) depends on traitlets (>=4.2)
   1: fact: ipython (6.5.0) depends on simplegeneric (>0.8)
   1: fact: ipython (6.5.0) depends on pickleshare (*)
   1: fact: ipython (6.5.0) depends on decorator (*)
   1: fact: ipython (6.5.0) depends on jedi (>=0.10)
   1: selecting ipython (6.5.0)
   1: derived: jedi (>=0.10)
   1: derived: decorator (*)
   1: derived: pickleshare (*)
   1: derived: simplegeneric (>0.8)
   1: derived: traitlets (>=4.2)
   1: derived: prompt-toolkit (>=1.0.15,<2.0.0)
   1: derived: backcall (*)
   1: derived: pexpect (*)
   1: derived: appnope (*)
   1: derived: colorama (*)
PyPI: 9 packages found for jedi >=0.10
PyPI: No release information found for decorator-3.4.1, skipping
PyPI: No release information found for decorator-4.0.8, skipping
PyPI: 20 packages found for decorator *
PyPI: 12 packages found for pickleshare *
PyPI: 1 packages found for simplegeneric >0.8
PyPI: No release information found for traitlets-4.0.0.dev, skipping
PyPI: 7 packages found for traitlets >=4.2
PyPI: 1 packages found for prompt-toolkit >=1.0.15,<2.0.0
PyPI: 1 packages found for backcall *
PyPI: No release information found for pexpect-0.97, skipping
PyPI: No release information found for pexpect-2.01, skipping
PyPI: 15 packages found for pexpect *
PyPI: 8 packages found for appnope *
PyPI: 38 packages found for colorama *
PyPI: Getting info for simplegeneric (0.8.1) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading sdist: simplegeneric-0.8.1.zip
   1: selecting simplegeneric (0.8.1)
PyPI: Getting info for prompt-toolkit (1.0.15) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading sdist: prompt_toolkit-1.0.15.tar.gz
   1: fact: prompt-toolkit (1.0.15) depends on six (>=1.9.0)
   1: fact: prompt-toolkit (1.0.15) depends on wcwidth (*)
   1: selecting prompt-toolkit (1.0.15)
   1: derived: wcwidth (*)
   1: derived: six (>=1.9.0)
PyPI: 9 packages found for wcwidth *
PyPI: Getting info for backcall (0.1.0) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading sdist: backcall-0.1.0.tar.gz
   1: selecting backcall (0.1.0)
PyPI: Getting info for pytest-cov (2.6.0) from PyPI
   1: fact: pytest-cov (2.6.0) depends on coverage (>=4.4)
   1: fact: pytest-cov (2.6.0) depends on pytest (>=2.9)
   1: selecting pytest-cov (2.6.0)
   1: derived: coverage (>=4.4)
PyPI: No release information found for coverage-2.5, skipping
PyPI: No release information found for coverage-2.6, skipping
PyPI: No release information found for coverage-2.75, skipping
PyPI: No release information found for coverage-2.76, skipping
PyPI: No release information found for coverage-2.77, skipping
PyPI: No release information found for coverage-2.78, skipping
PyPI: No release information found for coverage-2.8, skipping
PyPI: No release information found for coverage-2.80, skipping
PyPI: No release information found for coverage-2.85, skipping
PyPI: 8 packages found for coverage >=4.4
PyPI: Getting info for psycopg2-binary (2.7.5) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading sdist: psycopg2-binary-2.7.5.tar.gz
PyPI: Downloading wheel: psycopg2_binary-2.7.5-cp27-cp27m-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
   1: selecting psycopg2-binary (2.7.5)
PyPI: Getting info for six (1.11.0) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading wheel: six-1.11.0-py2.py3-none-any.whl
   1: selecting six (1.11.0)
PyPI: Getting info for chardet (3.0.4) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading wheel: chardet-3.0.4-py2.py3-none-any.whl
   1: selecting chardet (3.0.4)
PyPI: Getting info for idna (2.7) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading wheel: idna-2.7-py2.py3-none-any.whl
   1: selecting idna (2.7)
PyPI: Getting info for urllib3 (1.23) from PyPI
   1: selecting urllib3 (1.23)
PyPI: Getting info for pyjwt (1.6.4) from PyPI
   1: selecting pyjwt (1.6.4)
PyPI: Getting info for isort (4.3.4) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading sdist: isort-4.3.4.tar.gz
   1: selecting isort (4.3.4)
PyPI: Getting info for pytest-django (3.4.3) from PyPI
   1: fact: pytest-django (3.4.3) depends on pytest (>=3.6)
   1: selecting pytest-django (3.4.3)
   1: derived: pytest (>=3.6)
PyPI: Getting info for traitlets (4.3.2) from PyPI
   1: fact: traitlets (4.3.2) depends on decorator (*)
   1: fact: traitlets (4.3.2) depends on ipython-genutils (*)
   1: fact: traitlets (4.3.2) depends on six (*)
   1: selecting traitlets (4.3.2)
   1: derived: ipython-genutils (*)
PyPI: 2 packages found for ipython-genutils *
PyPI: Getting info for ipython-genutils (0.2.0) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading wheel: ipython_genutils-0.2.0-py2.py3-none-any.whl
   1: selecting ipython-genutils (0.2.0)
PyPI: Getting info for appnope (0.1.0) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading wheel: appnope-0.1.0-py2.py3-none-any.whl
   1: selecting appnope (0.1.0)
PyPI: Getting info for coverage (4.5.1) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading sdist: coverage-4.5.1.tar.gz
PyPI: Downloading wheel: coverage-4.5.1-cp26-cp26m-macosx_10_10_x86_64.whl
   1: selecting coverage (4.5.1)
PyPI: Getting info for urwid (2.0.1) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading sdist: urwid-2.0.1.tar.gz
   1: selecting urwid (2.0.1)
PyPI: Getting info for mock (2.0.0) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading wheel: mock-2.0.0-py2.py3-none-any.whl
   1: fact: mock (2.0.0) depends on pbr (>=0.11)
   1: fact: mock (2.0.0) depends on six (>=1.9)
   1: selecting mock (2.0.0)
   1: derived: pbr (>=0.11)
PyPI: 33 packages found for pbr >=0.11
PyPI: Getting info for certifi (2018.10.15) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading wheel: certifi-2018.10.15-py2.py3-none-any.whl
   1: selecting certifi (2018.10.15)
PyPI: Getting info for jedi (0.13.1) from PyPI
   1: fact: jedi (0.13.1) depends on parso (>=0.3.0)
   1: selecting jedi (0.13.1)
   1: derived: parso (>=0.3.0)
PyPI: 2 packages found for parso >=0.3.0
PyPI: Getting info for parso (0.3.1) from PyPI
PyPI: No dependencies found, downloading archives
   1: derived: more-itertools (>=4.0.0)
   1: derived: atomicwrites (>=1.0)
   1: derived: pluggy (>=0.7)
PyPI: No release information found for py-0.8.0-alpha2, skipping
PyPI: No release information found for py-0.9.0, skipping
PyPI: No release information found for py-1.4.32.dev1, skipping
PyPI: 6 packages found for py >=1.5.0
PyPI: 3 packages found for attrs >=17.4.0
PyPI: 5 packages found for more-itertools >=4.0.0
PyPI: 5 packages found for atomicwrites >=1.0
PyPI: 2 packages found for pluggy >=0.7
PyPI: Getting info for pluggy (0.8.0) from PyPI
   1: selecting pluggy (0.8.0)
PyPI: Getting info for attrs (18.2.0) from PyPI
   1: selecting attrs (18.2.0)
PyPI: Getting info for more-itertools (4.3.0) from PyPI
   1: fact: more-itertools (4.3.0) depends on six (>=1.0.0,<2.0.0)
   1: selecting more-itertools (4.3.0)
PyPI: Getting info for atomicwrites (1.2.1) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading wheel: atomicwrites-1.2.1-py2.py3-none-any.whl
   1: selecting atomicwrites (1.2.1)
PyPI: Getting info for py (1.7.0) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading wheel: py-1.7.0-py2.py3-none-any.whl
   1: selecting py (1.7.0)
PyPI: Getting info for pexpect (4.6.0) from PyPI
   1: fact: pexpect (4.6.0) depends on ptyprocess (>=0.5)
   1: selecting pexpect (4.6.0)
   1: derived: ptyprocess (>=0.5)
PyPI: 4 packages found for ptyprocess >=0.5
PyPI: Getting info for ptyprocess (0.6.0) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading wheel: ptyprocess-0.6.0-py2.py3-none-any.whl
   1: selecting ptyprocess (0.6.0)
PyPI: Getting info for django (2.1.2) from PyPI
   1: fact: django (2.1.2) depends on pytz (*)
   1: selecting django (2.1.2)
PyPI: Getting info for black (18.9b0) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading sdist: black-18.9b0.tar.gz
   1: fact: black (18.9b0) depends on click (>=6.5)
   1: fact: black (18.9b0) depends on attrs (>=17.4.0)
   1: fact: black (18.9b0) depends on appdirs (*)
   1: fact: black (18.9b0) depends on toml (>=0.9.4)
   1: selecting black (18.9b0)
   1: derived: toml (>=0.9.4)
   1: derived: appdirs (*)
   1: derived: click (>=6.5)
PyPI: 3 packages found for toml >=0.9.4
PyPI: 7 packages found for appdirs *
PyPI: 5 packages found for click >=6.5
PyPI: Getting info for toml (0.10.0) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading wheel: toml-0.10.0-py2.py3-none-any.whl
   1: selecting toml (0.10.0)
PyPI: Getting info for click (7.0) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading wheel: Click-7.0-py2.py3-none-any.whl
   1: selecting click (7.0)
PyPI: Getting info for appdirs (1.4.3) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading wheel: appdirs-1.4.3-py2.py3-none-any.whl
   1: selecting appdirs (1.4.3)
PyPI: Getting info for docutils (0.14) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading sdist: docutils-0.14.tar.gz
PyPI: Downloading wheel: docutils-0.14-py2-none-any.whl
   1: selecting docutils (0.14)
PyPI: Getting info for decorator (4.3.0) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading wheel: decorator-4.3.0-py2.py3-none-any.whl
   1: selecting decorator (4.3.0)
PyPI: Getting info for restructuredtext-lint (1.1.3) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading sdist: restructuredtext_lint-1.1.3.tar.gz
   1: fact: restructuredtext-lint (1.1.3) depends on docutils (>=0.11,<1.0)
   1: selecting restructuredtext-lint (1.1.3)
PyPI: Getting info for protobuf (3.6.1) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading wheel: protobuf-3.6.1-py2.py3-none-any.whl
   1: fact: protobuf (3.6.1) depends on six (>=1.9)
   1: selecting protobuf (3.6.1)
PyPI: Getting info for pbr (5.0.0) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading wheel: pbr-5.0.0-py2.py3-none-any.whl
   1: selecting pbr (5.0.0)
PyPI: Getting info for colorama (0.4.0) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading wheel: colorama-0.4.0-py2.py3-none-any.whl
   1: selecting colorama (0.4.0)
PyPI: Getting info for faker (0.9.2) from PyPI
   1: fact: faker (0.9.2) depends on text-unidecode (1.2)
   1: fact: faker (0.9.2) depends on six (>=1.10)
   1: fact: faker (0.9.2) depends on python-dateutil (>=2.4)
   1: selecting faker (0.9.2)
   1: derived: python-dateutil (>=2.4)
   1: derived: text-unidecode (1.2)
PyPI: No release information found for python-dateutil-0.1, skipping
PyPI: No release information found for python-dateutil-0.3, skipping
PyPI: No release information found for python-dateutil-0.4, skipping
PyPI: No release information found for python-dateutil-0.5, skipping
PyPI: No release information found for python-dateutil-1.0, skipping
PyPI: No release information found for python-dateutil-1.1, skipping
PyPI: No release information found for python-dateutil-1.2, skipping
PyPI: No release information found for python-dateutil-2.0, skipping
PyPI: 13 packages found for python-dateutil >=2.4
PyPI: 1 packages found for text-unidecode 1.2
PyPI: Getting info for text-unidecode (1.2) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading wheel: text_unidecode-1.2-py2.py3-none-any.whl
   1: selecting text-unidecode (1.2)
PyPI: Getting info for python-dateutil (2.7.3) from PyPI
   1: fact: python-dateutil (2.7.3) depends on six (>=1.5)
   1: selecting python-dateutil (2.7.3)
PyPI: Getting info for grpcio-reflection (1.15.0) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading sdist: grpcio-reflection-1.15.0.tar.gz
   1: fact: grpcio-reflection (1.15.0) depends on protobuf (>=3.6.0)
   1: fact: grpcio-reflection (1.15.0) depends on grpcio (>=1.15.0)
   1: selecting grpcio-reflection (1.15.0)
   1: derived: grpcio (>=1.15.0)
PyPI: Getting info for grpcio (1.15.0) from PyPI
   1: fact: grpcio (1.15.0) depends on enum34 (>=1.0.4)
   1: fact: grpcio (1.15.0) depends on futures (>=2.2.0)
   1: fact: grpcio (1.15.0) depends on six (>=1.5.2)
   1: selecting grpcio (1.15.0)
   1: derived: futures (>=2.2.0)
   1: derived: enum34 (>=1.0.4)
PyPI: 10 packages found for futures >=2.2.0
PyPI: 8 packages found for enum34 >=1.0.4
PyPI: Getting info for enum34 (1.1.6) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading sdist: enum34-1.1.6.tar.gz
PyPI: Downloading wheel: enum34-1.1.6-py2-none-any.whl
   1: selecting enum34 (1.1.6)
PyPI: Getting info for futures (3.2.0) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading sdist: futures-3.2.0.tar.gz
PyPI: Downloading wheel: futures-3.2.0-py2-none-any.whl
   1: fact: futures (3.2.0) requires Python >=2.6, <3
   1: derived: not futures (3.2.0)
PyPI: Getting info for futures (3.1.1) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading sdist: futures-3.1.1.tar.gz
PyPI: Downloading wheel: futures-3.1.1-py2-none-any.whl
   1: selecting futures (3.1.1)
PyPI: Getting info for stevedore (1.29.0) from PyPI
   1: fact: stevedore (1.29.0) depends on six (>=1.10.0)
   1: fact: stevedore (1.29.0) depends on pbr (>=2.0.0,<2.1.0 || >2.1.0)
   1: selecting stevedore (1.29.0)
PyPI: Getting info for cryptography (2.3.1) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading sdist: cryptography-2.3.1.tar.gz
   1: fact: cryptography (2.3.1) depends on idna (>=2.1)
   1: fact: cryptography (2.3.1) depends on asn1crypto (>=0.21.0)
   1: fact: cryptography (2.3.1) depends on six (>=1.4.1)
   1: fact: cryptography (2.3.1) depends on cffi (>=1.7,<1.11.3 || >1.11.3)
   1: selecting cryptography (2.3.1)
   1: derived: cffi (>=1.7,<1.11.3 || >1.11.3)
   1: derived: asn1crypto (>=0.21.0)
PyPI: 11 packages found for cffi >=1.7,<1.11.3 || >1.11.3
PyPI: 5 packages found for asn1crypto >=0.21.0
PyPI: Getting info for asn1crypto (0.24.0) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading wheel: asn1crypto-0.24.0-py2.py3-none-any.whl
   1: selecting asn1crypto (0.24.0)
PyPI: Getting info for cffi (1.11.5) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading sdist: cffi-1.11.5.tar.gz
   1: fact: cffi (1.11.5) depends on pycparser (*)
   1: selecting cffi (1.11.5)
   1: derived: pycparser (*)
PyPI: 19 packages found for pycparser *
PyPI: Getting info for pycparser (2.19) from PyPI
PyPI: No dependencies found, downloading archives
PyPI: Downloading sdist: pycparser-2.19.tar.gz
   1: selecting pycparser (2.19)
   1: Version solving took 18.244 seconds.
   1: Tried 1 solutions.


Package operations: 68 installs, 0 updates, 0 removals, 2 skipped

Writing lock file

  - Installing enum34 (1.1.6)
  - Installing futures (3.1.1)
  - Installing pycparser (2.19)
  - Installing six (1.11.0)
  - Installing asn1crypto (0.24.0)
  - Installing atomicwrites (1.2.1)
  - Installing attrs (18.2.0)
  - Installing cffi (1.11.5)
  - Skipping colorama (0.4.0) Not needed for the current environment
  - Installing decorator (4.3.0)
  - Installing docutils (0.14)
  - Installing grpcio (1.15.0)
  - Installing idna (2.7)
  - Installing ipython-genutils (0.2.0)
  - Installing more-itertools (4.3.0)
  - Installing parso (0.3.1)
  - Installing pbr (5.0.0)
  - Installing pluggy (0.8.0)
  - Installing protobuf (3.6.1)
  - Installing ptyprocess (0.6.0)
  - Installing py (1.7.0)
  - Installing python-dateutil (2.7.3)
  - Installing pytz (2018.5)
  - Installing text-unidecode (1.2)
  - Installing wcwidth (0.1.7)
  - Installing appdirs (1.4.3)
  - Skipping appnope (0.1.0) Not needed for the current environment
  - Installing backcall (0.1.0)
  - Installing certifi (2018.10.15)
  - Installing chardet (3.0.4)
  - Installing click (7.0)
  - Installing coverage (4.5.1)
  - Installing cryptography (2.3.1)
  - Installing django (2.1.2)
  - Installing faker (0.9.2)
  - Installing grpcio-reflection (1.15.0)
  - Installing jedi (0.13.1)
  - Installing mock (2.0.0)
  - Installing pexpect (4.6.0)
  - Installing pickleshare (0.7.5)
  - Installing prompt-toolkit (1.0.15)
  - Installing pygments (2.2.0)
  - Installing pyjwt (1.6.4)
  - Installing pytest (3.9.1)
  - Installing restructuredtext-lint (1.1.3)
  - Installing simplegeneric (0.8.1)
  - Installing stevedore (1.29.0)
  - Installing toml (0.10.0)
  - Installing traitlets (4.3.2)
  - Installing typed-ast (1.1.0)
  - Installing urllib3 (1.23)
  - Installing urwid (2.0.1)
  - Installing black (18.9b0)
  - Installing dj-database-url (0.5.0)
  - Installing django-grpc (0.1.1 7a5009d)
  - Installing doc8 (0.8.0)
  - Installing envparse (0.2.0)
  - Installing factory-boy (2.11.1)
  - Installing ipython (6.5.0)
  - Installing isort (4.3.4)
  - Installing mypy (0.620)
  - Installing mypy-protobuf (1.7)
  - Installing pillow (5.3.0)
  - Installing psycopg2-binary (2.7.5)
  - Installing pudb (2018.1)
  - Installing pyflakes (2.0.0)
  - Installing pytest-cov (2.6.0)
  - Installing pytest-django (3.4.3)
  - Installing pytest-spec (1.1.0)
  - Installing requests (2.19.1)
                                                       
[ValueError]                            
No file/folder found for package mos-locate-service  
                                                       
Exception trace:
 /home/circleci/.poetry/lib/poetry/_vendor/py3.7/cleo/application.py in run() at line 94
   status_code = self.do_run(input_, output_)
 /home/circleci/.poetry/lib/poetry/console/application.py in do_run() at line 88
   return super(Application, self).do_run(i, o)
 /home/circleci/.poetry/lib/poetry/_vendor/py3.7/cleo/application.py in do_run() at line 197
   status_code = command.run(input_, output_)
 /home/circleci/.poetry/lib/poetry/console/commands/command.py in run() at line 77
   return super(BaseCommand, self).run(i, o)
 /home/circleci/.poetry/lib/poetry/_vendor/py3.7/cleo/commands/base_command.py in run() at line 146
   status_code = self.execute(input_, output_)
 /home/circleci/.poetry/lib/poetry/_vendor/py3.7/cleo/commands/command.py in execute() at line 107
   return self.handle()
 /home/circleci/.poetry/lib/poetry/console/commands/install.py in handle() at line 67
   builder = SdistBuilder(self.poetry, NullEnv(), NullIO())
 /home/circleci/.poetry/lib/poetry/masonry/builders/builder.py in __init__() at line 35
   includes=self._package.include,
 /home/circleci/.poetry/lib/poetry/masonry/utils/module.py in __init__() at line 51
   raise ValueError("No file/folder found for package {}".format(name))

install [--no-dev] [--dry-run] [-E|--extras EXTRAS] [--develop DEVELOP]

Makefile:41: recipe for target 'install' failed
make: *** [install] Error 1
Exited with code 2
```

I have tried a number of things including changing the directory where the repo is checked out, changing the package name in `pyproject.toml`, updating the lock file, etc to no avail. I'm not having this issue on MacOS, though...

**Comment by shabbyrobe** [comment]
I'm experiencing this too. After reading the source for `masonry/utils/module.py`, I managed to convince poetry to continue by adding a file to my `src` directory with the name of my project, i.e. given the `pyproject.toml`:

    [tool.poetry]
    name = "pants"

I added an empty file called `<proj>/src/pants.py`, and now I am unblocked.

**Comment by pikeas** [comment]
Same issue here, temporarily solved by adding:

```RUN mkdir my_app && touch my_app/__init__.py```

**Comment by sdispater** [comment]
Just so I understand correctly and make a proper fix: your project does not follow the structure expected by Poetry and, so, you do not rely on the packaging feature of Poetry. Is that correct?

If that's the case, the recent change to `install` (installing the current project in editable mode) will break your current workflow.

I will make a quick fix and get back to you.

**Comment by pikeas** [comment]
@sdispater I believe this is correct. My project is an application rather than a library, so I'm using Poetry only to manage our dependency lifecycle and not for publishing.

I'll add that the poetry docs for install don't say anything about installing the user's library:

```
The install command reads the pyproject.toml file from the current directory, resolves the dependencies, and installs them.
```

**Comment by pikeas** [comment]
Actually, I just compared by project structure to the docs and they do match:

```
poetry-demo
├── pyproject.toml
├── README.rst
├── poetry_demo
│   └── __init__.py
```

My Dockerfile does the following:

```
COPY pyproject.toml .
COPY pyproject.lock .

# https://github.com/sdispater/poetry/issues/512
RUN mkdir poetry_demo && touch poetry_demo/__init__.py

RUN poetry install
COPY poetry_demo/ ./poetry_demo
```

**Comment by jacebrowning** [comment]
@sdispater Another use case: We use Poetry to manage a [cookiecutter](https://github.com/audreyr/cookiecutter) template, which will never have a package of its own. Only the generated projects contain valid Python packages.

Being able to set `packages = false` (or have this detected automatically) would help us out.

**Comment by lmk-cc** [comment]
Hey @sdispater thanks for the quick action on this! I was just wondering, what's the best way to install poetry from github as I'd like to get my CI builds working again :-)

**Comment by pikeas** [comment]
@sdispater Thanks for the quick commit from me as well! One quick note, the commit comments are inaccurate: I'm hitting this failure with a project that *does* match the recommended layout.

**Comment by sdispater** [comment]
@lmk-cc I just published release `0.12.3` with this fixed included.

@pikeas I am not sure why this happens in your case. Could you post a gist of your `pyproject.toml`?

**Comment by lmk-cc** [comment]
Thanks for the fix @sdispater 

For the record, this works out as a nice way to use `poetry` from within circle:

```
      - run:
          name: install poetry
          command: |
            curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
            echo 'export PATH="$HOME/.poetry/bin:$PATH"' >> $BASH_ENV

      - run:
          name: install dependencies
          command: |
            poetry config settings.virtualenvs.in-project true
            make install
```

(The script at `$BASH_ENV` is sourced by all subsequent build steps)

**Comment by krnr** [comment]
> However, I think we should have a way of telling Poetry that we don't want the project to be built.

@sdispater this will be indispensable feature! just think of all the projects that are developed in containers

- you copy only project.toml first to make a cached layer of only deps
- you don't have your project code at all in the container
- poetry tries to install deps and hitting the error when sees an empty folder of your project

With the things how they are now you enforce people to copy the whole project into the container, which in turn leads to dependencies being installed EVERY single time a code changed. Which is obviously not what we want as developers.

Thank you for such prompt reaction and a really nice package.

**Comment by fbergroth** [comment]
@krnr this seems to be fixed in 0.12.3. Here's the relevant part of my Dockerfile:
```
RUN POETRY_VERSION=0.12.3 curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
ENV PATH=/root/.poetry/bin:$PATH
RUN poetry config settings.virtualenvs.create false
WORKDIR /app
COPY pyproject.toml ./
COPY poetry.lock ./
RUN poetry install --no-ansi --no-interaction
```

**Comment by sdispater** [comment]
Closing since it has been fixed in the `0.12.3` release.
